src/pudl/convert/censusdp1tract_to_sqlite.py

```diff
@@ -0,0 +1,99 @@
+"""
+Convert the US Census DP1 ESRI GeoDatabase into an SQLite Database.
+
+This is a thin wrapper around GDAL's ogr2ogr utility, which does the actual
+conversion. The code in this module tells that utility where to find the input
+data in the PUDL Datastore, and where to output the SQLite DB, alongside our
+other SQLite Databases (ferc1.sqlite and pudl.sqlite)
+
+"""
+
+import argparse
+import logging
+import os
+import subprocess  # nosec: B404
+import sys
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import coloredlogs
+
+import pudl
+from pudl.workspace.datastore import Datastore
+
+logger = logging.getLogger(__name__)
+
+
+def censusdp1tract_to_sqlite(pudl_settings=None, year=2010):
+    """
+    Use GDAL's ogr2ogr utility to convert the Census DP1 GeoDB to an SQLite DB.
+
+    The Census DP1 GeoDB is read from the datastore, unzipped into a temporary
+    directory, and then converted to SQLite using an external process. The
+    resulting SQLite DB is put in the PUDL output directory alongside the ferc1
+    and pudl SQLite databases.
+
+    Args:
+        pudl_settings (dict): A PUDL settings dictionary.
+        year (int): Year of Census data to extract (currently must be 2010)
+
+    Returns:
+        None
+
+    """
+    if pudl_settings is None:
+        pudl_settings = pudl.workspace.setup.get_defaults()
+    ds = Datastore(local_cache_path=pudl_settings["data_dir"])
+
+    # Avoid executing any random program that happens to be named ogr2ogr
+    # If we're in a conda environment, use the ogr2ogr there
+    # Otherwise assume it's where Ubuntu installs it /usr/bin/ogr2ogr
+    prefix_dir = os.environ.get("CONDA_PREFIX", "/usr")
+    ogr2ogr = prefix_dir + "/bin/ogr2ogr"
```

Comment:
Not the world's most robust solution, but I think it will serve our purposes in conda, Docker, and CI environments fine.

Reply:
why would there be another program that was randomly named ogr2ogr in the env?

Reply:
i'm not sure why you went with this odd non-import

Reply:
Tried to explain this more in the module-level docstring.